.editorconfig

```diff
@@ -0,0 +1,268 @@
+root = true
```

Comment:
I think this file should not be here, it looks like a config file for your editor ;-)

Reply:
I don't know which editor you are using, but I thought it might be helpful for contributors to have an .editorconfig with the formatting rules available - with the default formatting in IntelliJ there would be changes in every single line.

Reply:
I understand, my idea here is to use the https://github.com/coveooss/fmt-maven-plugin plugin to format the code.

Reply:
I didn't see that this plugin was present at first. I'll format everything with that one and also added a note in the CONTRIBUTING document.